### PR TITLE
My 21 footprint download tool

### PR DESCRIPTION
**Pull request by nicreask**

### CODE REVIEW

**Comment by nicreask** [comment]
I forgot to update the README - that is done now. It should be complete pending review.

File: tools/get_metryc_footprint.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    terrain_map = {'full_terrain_gust': 'FT',
+                   'open_water': 'OW',
+                   'open_terrain': 'OT'}
+    args['terrain_correction'] = terrain_map[args['terrain_correction']]
+
+    if args['format'] == 'geotiff':
+        suffix = 'tiff'
+    else:
+        suffix = args['format']
```
Comment (jasonre0) [inline tools/get_metryc_footprint.py:24]: saves some lines if you move suffix and template above these blocks and set defaults there.

File: tools/get_metryc_footprint.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    if args.format not in ['geotiff', 'geojson']:
+        print('Unsupported output format.')
+        parser.print_help()
+        return 1
+
+    min_lat = args.bbox[0]
+    max_lat = args.bbox[1]
+    min_lon = args.bbox[2]
+    max_lon = args.bbox[3]
```
Comment (jasonre0) [inline tools/get_metryc_footprint.py:86]: consider 

```
min_lat, max_lat, min_lon, max_lon = args.bbox
```

**Review by jasonre0 — APPROVED** [review]
lgtm, some minor comments

**Pull request merged**